onnxscript/rewriter/pattern.py

```diff
@@ -467,7 +462,7 @@ def matches_node(self, node: ir.Node, model: ir.Model) -> MatchResult:
             return MatchResult.FAIL()
         if not self.op.matches(node.op_type):
             return MatchResult.FAIL()
-        match = MatchResult([])
+        match = MatchResult(success=True)
```

Comment:
If the usage is to assume it's a success in the begining, should we set a default to MatchResult?